Content/.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": 1,
+    "isRoot": true,
+    "tools": {
+      "safe.tool": {
+        "version": "0.0.6",
+        "commands": [
+          "safe"
+        ]
```

Comment:
Any plans to add paket and fake as local dotnet tools for the template?

Reply:
yes, eventually